### PR TITLE
`sov-cli` can submit transactions to `sov-sequencer` and can trigger publishing batch

### PR DESCRIPTION
**Pull request by citizen-stig**
# Description

* Introduces 2 new sov-cli commands: `submit_tx` and `publish_batch` 
* Fixes batch serialization from flattening to using `borsh`
* Fixes parameters for `state.submitPayForBlob` celestia endpoint
* Fixes address display for celestia
* Using Same celestia address that is used by RPC node
* Celestia adapter handles errors better

## Linked Issues
- Related to #391 

## Testing
Automated tests added, including mocking responses from Celestia node

## Docs

Demo rollup documentation has been updated, as incorrect private key was used in Makefile, that lead to transfer and mint transactions were unusable because of different token address

## Following work

This pull request offloaded many detected issues:

* https://github.com/Sovereign-Labs/sovereign-sdk/issues/478
* https://github.com/Sovereign-Labs/sovereign-sdk/issues/473
* https://github.com/Sovereign-Labs/sovereign-sdk/issues/472
* https://github.com/Sovereign-Labs/sovereign-sdk/issues/469
* https://github.com/Sovereign-Labs/sovereign-sdk/issues/465

### CODE REVIEW

File: adapters/celestia/src/da_service.rs [hunk 1]
```diff
@@ -106,6 +107,10 @@ fn default_max_response_size() -> u32 {
     1024 * 1024 * 100 // 100 MB
 }
 
+fn default_request_timeout_ms() -> u64 {
+    10_000
+}
```
Comment (bkolad) [inline adapters/celestia/src/da_service.rs:112]: let's make it `const`

File: adapters/celestia/src/da_service.rs [hunk 2]
```diff
@@ -106,6 +107,10 @@ fn default_max_response_size() -> u32 {
     1024 * 1024 * 100 // 100 MB
```
Comment (bkolad) [inline adapters/celestia/src/da_service.rs:107]: let's make it const

File: adapters/celestia/src/da_service.rs (hunk 1 above)
Comment (citizen-stig) [reply adapters/celestia/src/da_service.rs:112]: Why?

I made it this way, as ground work for https://github.com/Sovereign-Labs/sovereign-sdk/issues/478

File: adapters/celestia/src/da_service.rs [hunk 3]
```diff
@@ -126,9 +131,12 @@ impl DaService for CelestiaService {
                     .parse()
                     .unwrap(),
             );
+
             jsonrpsee::http_client::HttpClientBuilder::default()
```
Comment (bkolad) [inline adapters/celestia/src/da_service.rs:135]: Is it possible to separate the `client` building logic (and tx sending) to a separate file? This way we will be able to reuse it more easily.

**Review by bkolad — APPROVED** [review]

File: adapters/celestia/src/da_service.rs (hunk 1 above)
Comment (bkolad) [reply adapters/celestia/src/da_service.rs:112]: Ok I thought it will be a hardcoded value.

**Review by dubbelosix — APPROVED** [review]
LGTM!

**Pull request merged**